front/src/app/services/user/user.service.ts

```diff
@@ -40,9 +36,13 @@ export class UserService {
         id: Object.freeze(userData.id),
         email: Object.freeze(userData.email),
         username: Object.freeze(userData.username),
+        game_counter: userData.game_counter,
+        win_dict: userData.win_dict,
+        time_played: userData.time_played,
         color_config: userData.color_config,
-        game_settings: Object.freeze(userData.game_settings),
```

Comment:
Why is the freeze unfrozen?

Reply:
because the Object.freeze(...) is supposed to return a read-only object. But games_settings can be changed.